sass/site/site.scss

```diff
@@ -158,7 +158,7 @@ $paddingNormal: 20px;
 // --------------------------------------------------
 .navbar {
   margin-bottom: 0px;
-  border-top: 3px solid $blueDark;
+  border-top: 3px solid #0d94d2; //cerner blue
```

Comment:
Go ahead and add $cernerBlue and $cernerGreen variables to the variables.scss so those color variables can be used  in the site styles.